kolibri/plugins/user/assets/src/modules/pluginModule.js

```diff
@@ -30,6 +31,7 @@ export default {
     },
     setFacilityId(store, { facilityId }) {
       store.commit('SET_FACILITY_ID', facilityId);
+      store.dispatch('getFacilityConfig', facilityId);
```

Comment:
Should return here so that we can wait on resolution of this async behaviour from the wrapping action.